### PR TITLE
Stein VI docs

### PR DESCRIPTION
**Pull request by OlaRonning**
Expose documentation for SteinVI. Related #1297 #1298.

### CODE REVIEW

File: docs/source/contrib.rst [hunk 1]
```diff
@@ -9,3 +9,66 @@ Nested Sampling
...
+- `LinearKernel`
+- `RandomFeatureKernel`
+- `MixtureKernel`
+- `PrecondMatrixKernel`
+- `HessianPrecondMatrix`
+- `GraphicalKernel`
+
+For example, usage see:
+
+- The `Bayesian neural network example <https://num.pyro.ai/examples/stein_bnn.html>`_
```
Comment (OlaRonning) [inline docs/source/contrib.rst:49]: @fehiepsi can you check the url?

Comment (fehiepsi) [reply docs/source/contrib.rst:49]: It is https://num.pyro.ai/en/latest/examples/stein_bnn.html

**Review by fehiepsi — DISMISSED** [review]
LGTM, thanks Ola!

**Comment by OlaRonning** [comment]
Thanks, @fehiepsi; I've updated to the correct example URL now.

**Comment by fehiepsi** [comment]
Hi @OlaRonning, could you help me ping jax to `0.2.27` or add `@pytest.mark.xfail(reason="jax 0.2.28 breaks provenance logic.")` to the failing test? I'll try to resolve the issue in the weekend.

**Comment by OlaRonning** [comment]
Sure, np. I’ll fix it once kids are in bed.

**Review by fehiepsi — APPROVED** [review]
Thanks, Ola!

**Pull request merged**